README.md

```diff
@@ -81,28 +52,40 @@ For more details, check out the [echopype documentation](https://echopype.readth
 and watch the [echopype talk](https://www.youtube.com/watch?v=qboH7MyHrpU) 
 at SciPy 2019 for background, discussions and a quick demo!
 
+## Documentation
+
+Learn more about echopype in the official documentation at https://echopype.readthedocs.io
+
+
+## Contributing
+
+You can find information about how to contribute to echopype at our [Contributing Page](https://echopype.readthedocs.io/en/latest/contributing.html).
+
 
 ## Echopype doesn't run on your data?
 
 Please report any bugs by [creating issues on GitHub](https://medium.com/nyc-planning-digital/writing-a-proper-github-issue-97427d62a20f).
+
 [Pull requests](https://jarednielsen.com/learn-git-fork-pull-request/) are always welcome!
 
 
 Contributors
 ------------
 
 [Wu-Jung Lee](http://leewujung.github.io) (@leewujung) leads this project
-and along with [Kavin Nguyen](https://github.com/ngkavin) (@ngkavin)
-are primary developers of this package.
-[Valentina Staneva](https://escience.washington.edu/people/valentina-staneva/) (@valentina-s)
-and [Emilio Mayorga](https://www.apl.washington.edu/people/profile.php?last_name=Mayorga&first_name=Emilio) (@emiliom)
-provide consultation and also contribute to the development.
+and is the primary developer of this package together with 
+[Kavin Nguyen](https://github.com/ngkavin) (@ngkavin)
+and [Landung "Don" Setiawan](https://github.com/lsetiawan) (@lsetiawan).
+[Emilio Mayorga](https://www.apl.washington.edu/people/profile.php?last_name=Mayorga&first_name=Emilio) (@emiliom)
+and [Valentina Staneva](https://escience.washington.edu/people/valentina-staneva/) (@valentina-s)
+are also part of the development team.
```

Comment:
Can't quite wrap around my head how to phrase this. How about:
[Wu-Jung Lee](http://leewujung.github.io) (@leewujung) leads this project and together with Kavin Nguyen (@ngkavin), Landung "Don" Setiawan(@lsetiawan), and Imran Majeed (@imranmaj) are primary developers of this package. [Emilio Mayorga](https://www.apl.washington.edu/people/profile.php?last_name=Mayorga&first_name=Emilio) (@emiliom) and [Valentina Staneva](https://escience.washington.edu/people/valentina-staneva/) (@valentina-s) are also part of the development team.